app/src/main/java/org/simple/clinic/bloodsugar/history/BloodSugarHistoryScreen.kt

```diff
@@ -1,23 +1,34 @@
 package org.simple.clinic.bloodsugar.history
 
+import android.annotation.SuppressLint
 import android.content.Context
+import android.content.Intent
```

Comment:
Reword the commit message? Ex: `Implement UI action methods in BloodSugarHistoryScreen`